### ISSUE TITLE
WinForms TabControl does not expose TabItem controls

### ISSUE THREAD

**Issue by leind375** [dotnet/winforms#10408]
### .NET version

.NET 8

### Did it work in .NET Framework?

Yes

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

Yes, I believe it was working in .NET 6.

### Issue description

For a WinForms TabControl, the tab page is exposed through UIAutomation but the TabItem controls are missing so there is no way for automations to switch between tabs.

### Steps to reproduce

Create a WinForms app. Add a TabControl. By default it will have to pages. Run it. Walk the tree of the application. Tab page is there but no TabItem controls.

**Comment by Tanya-Solyanik** [comment]
@Olina-Zhang  - Could you please verify when this scenario regressed? Consider using https://learn.microsoft.com/en-us/dotnet/framework/ui-automation/navigate-among-ui-automation-elements-with-treewalker to build a sample

**Comment by LeafShi1** [comment]
> @Olina-Zhang - Could you please verify when this scenario regressed? Consider using https://learn.microsoft.com/en-us/dotnet/framework/ui-automation/navigate-among-ui-automation-elements-with-treewalker to build a sample

I tried to write an sample using [Example 2](https://learn.microsoft.com/en-us/dotnet/framework/ui-automation/navigate-among-ui-automation-elements-with-treewalker), in the .netFramwork 4.8.1 project, TreeWalker can obtain the tabitem,
![image](https://github.com/dotnet/winforms/assets/132890443/5799ba8d-45eb-4e6d-a86d-eeb157e3d169)

but In the .netcore project,  TreeWalker obtain the panel.
![image](https://github.com/dotnet/winforms/assets/132890443/60fa8aba-c378-44db-ae63-c1d3655c0480)

Sample(the 'UIAutomationClient.dll' and 'UIAutomationTypes.dll' need to be referenced, I am referencing the dlls from C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.8.1):
[WindowsFormsApp3.zip](https://github.com/dotnet/winforms/files/13543159/WindowsFormsApp3.zip)

**Comment by SimonZhao888** [comment]
When we use reverse lookup, TreeWalker can obtain the TabItem controls.
![image](https://github.com/dotnet/winforms/assets/133954995/62989f17-b2b3-438a-baac-6df50c3ed1b0)
![image](https://github.com/dotnet/winforms/assets/133954995/fa9b3f40-d6a2-4ae4-9064-bae37bcd13c0)
[WinFormsApp71.zip](https://github.com/dotnet/winforms/files/13655700/WinFormsApp71.zip)

@leind375, hope this can help you to solve your problem.

**Comment by leind375** [comment]
Adding a comment to keep the defect open. The workaround will help us short term but would still like to see this fixed so we can avoid all the overrides we'll need to implement on future UI.

**Comment by leind375** [comment]
The workaround above (using GetLastChild() and GetPreviousSibling() also has a defect - GetPreviousSibling is also returning children, not just siblings.

**Comment by SimonZhao888** [comment]
> The workaround above (using GetLastChild() and GetPreviousSibling() also has a defect - GetPreviousSibling is also returning children, not just siblings.

We need to modify the code appropriately, e.g. try to use TreeWalker(Condition condition), here is the example code：
![1](https://github.com/dotnet/winforms/assets/133954995/31080298-9824-45f1-bdc0-b1978104adf6)

**Comment by leind375** [comment]
@SimonZhao888 Yes, I can work with this for now. Would still like to see this fixed. Starting to add a lot of hacks in UIA code...
